WordPress/src/main/java/org/wordpress/android/util/LocaleManager.kt

```diff
@@ -29,37 +19,17 @@ object LocaleManager {
     private const val MIN_LANGUAGE_CODE_LENGTH = 2
     private const val MAX_LANGUAGE_CODE_LENGTH = 6
 
-    /**
-     * Activate the locale associated with the provided context.
-     *
-     * @param context The current context.
-     */
-    @JvmStatic
-    fun setLocale(context: Context): Context {
-        return updateResources(context, getLanguage(context))
-    }
-
-    /**
-     * Compare the language for the current context with another language.
-     *
-     * @param language The language to compare
-     * @return True if the languages are the same, else false
-     */
-    fun isSameLanguage(language: String): Boolean {
-        val newLocale = languageLocale(language)
-        return Locale.getDefault().toString() == newLocale.toString()
-    }
 
+    @Suppress("ForbiddenComment")
     /**
-     * If the user has selected a language other than the device default, return that
-     * language code, else just return the device default language code.
+     * This is simply a wrapper for the per-app language code.
+     * TODO: Remove this and directly call PerAppLocaleManager
```

Comment:
> Note that for now LocaleManager.getLanguage() simply calls PerAppLocaleManager to get the current language code. This was to avoid replacing all the calls to LocaleManager.getLanguage() throughout the app. I will address that in a subsequent PR because I didn't want to make this one any larger than it is.

Suggestion (💡): Let's consider removing this and directly call `PerAppLocaleManager` now, and before we merge [the feature branch](https://github.com/wordpress-mobile/WordPress-Android/pull/21538) into `trunk`, usages show only 7 placed, and few more via the wrapper, with some classes already being touched by this PR anyway. I think having `LocalManager` and `LocalManagerWrapper` totally free of any language related functionality before merging this whole solution to `trunk` is a good win that is worth pursuing, even if this PR gets a bit more bigger. 🤔

Reply:
I would really prefer to do this separately. One reason for this is that I'm working a short day today and I'll be AFK until January 3, so I'd prefer to have this feature branch merged before I'm gone. There's not really any user impact for keeping this as-is.

Reply:
Noted Nick, this is purely a suggestion, you know the schedule best and if you do need that merged and going live to users before you return January 3rd, then it makes total sense to me too! 👍